### PR TITLE
Fixes incorrect words, grammar, and missing code formatting for dotnet-embedding errors documentation

### PR DESCRIPTION
**Pull request by theyo**

### CODE REVIEW

File: docs/tools/dotnet-embedding/errors.md
Comment (conceptdev) [inline docs/tools/dotnet-embedding/errors.md]: Ideally these strings match exactly what is emitted in the error console. I need to verify whether apostrophes or backticks are used in the product. Same applies to all edited headings in this PR.

**Review by conceptdev — COMMENTED** [review]
The language updates in this PR are great, thanks. I cannot merge it yet as I need to verify the heading format changes (apostrophes to backticks) correctly match the output a user would see. Once that has been done I will either merge or comment further.

File: docs/tools/dotnet-embedding/errors.md
Comment (theyo) [reply docs/tools/dotnet-embedding/errors.md]: Yeah, I wasn't sure either. If the emitted strings include the single quotes, I would advocate for putting the single quotes back in and keep the back ticks to denote the placeholder. This would keep it consistent with both the emitted output and the placeholder formatting in other headings.

**Comment by dannyjr894** [comment]
Remove all tags on my account

**Comment by dannyjr894** [comment]
Is it from my account if not y m I seen em

**Comment by PRMerger9** [comment]
@theyo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by lobrien — APPROVED** [review]

**Comment by lobrien** [comment]
#assign @conceptdev

File: docs/tools/dotnet-embedding/errors.md
Comment (davidbritch) [reply docs/tools/dotnet-embedding/errors.md]: The source shows that apostrophes are correct:

```
throw ErrorHelper.CreateError (7, $"Could not get the sdk version for '{sdk}'");
```

**Comment by PRMerger8** [comment]
@davidbritch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by davidbritch — APPROVED** [review]
Thanks @theyo and sorry for the long delay with this!

**Pull request merged**
